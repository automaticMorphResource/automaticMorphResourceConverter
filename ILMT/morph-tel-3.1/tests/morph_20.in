1	ixi	unk	
1	sahajaM	unk	
1	.	unk	
