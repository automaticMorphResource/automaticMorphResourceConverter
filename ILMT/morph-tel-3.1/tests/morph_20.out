1	ixi	unk	<fs af='ixi,pn,fn,sg,3,d,0,0'>
2	sahajaM	unk	<fs af='sahajaM,n,,sg,,d,0,0'>
3	.	unk	<fs af='&dot;,punc,,,,,,'>
